docs/components/menu/index.html

```diff
@@ -0,0 +1,156 @@
+---
+title: Menu
+assets:
+  - helix-ui.css
+  - helix-ui.js
```

Comment:
Not sure of the state of the docs refactor, but we won't need the `assets` here anymore, right?

Reply:
good catch! removed